src/test/java/org/folio/rest/utils/ResourceClients.java

```diff
@@ -12,6 +12,10 @@ public static ResourceClient accountsPayCheckClient(String accountId) {
     return new ResourceClient("/accounts/" + accountId + "/check-pay");
   }
 
+  public static ResourceClient accountsWaiveCheckClient(String accountId) {
```

Comment:
Does it makes sense to change the URL's word ordering in this method's name?

Reply:
reordered